pages/common/jj.md

```diff
@@ -0,0 +1,25 @@
+# jj
+
+> Jujutsu, a version control system.
+> Some subcommands such as `log`, `desc`, `new`, `git`, etc. have their own usage documentation.
```

Comment:
Note all of these exist yet. If all these PRs go in close together in time, maybe that's fine. I don't know how the tldr folks feel about it.

Reply:
It's fine. They will be merged within a week of each other and nothing here is really mission critical.